pandas/tests/arrays/categorical/test_map.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+import pytest
+
+import pandas as pd
+from pandas import (
+    Categorical,
+    Index,
+    Series,
+)
+import pandas._testing as tm
+
+
+@pytest.fixture(params=[None, "ignore"])
+def na_action(request):
+    return request.param
+
+
+class TestMap:
```

Comment:
Nit: Could you keep the tests as stand alone functions?

Reply:
I've changed it.